cmd/pw_ws_broker/main.go

```diff
@@ -49,6 +49,18 @@ func main() {
 			Value:   "amqp://guest:guest@rabbitmq:5672/pw",
 			EnvVars: []string{"SOURCE"},
```

Comment:
Make this `RABBITMQ` to match the router?

Reply:
updated source to be rabbitmq, still accepts source for now as an alias for BC